remoting/tests/src/test/java/org/jboss/as/remoting/RemotingSubsystemTestCase.java

```diff
@@ -326,12 +328,14 @@ protected void setupController(ControllerInitializer controllerInitializer) {
             protected void addExtraServices(ServiceTarget target) {
                 //Needed for initialization of the RealmAuthenticationProviderService
                 AbsolutePathService.addService(ServerEnvironment.CONTROLLER_TEMP_DIR, new File("target/temp" + System.currentTimeMillis()).getAbsolutePath(), target);
-                target.addService(IOServices.WORKER.append("default"), new WorkerService(Xnio.getInstance().createWorkerBuilder().setWorkerIoThreads(2)))
-                        .setInitialMode(ServiceController.Mode.ACTIVE)
-                        .install();
-                target.addService(IOServices.WORKER.append("default-remoting"), new WorkerService(Xnio.getInstance().createWorkerBuilder().setWorkerIoThreads(2)))
-                        .setInitialMode(ServiceController.Mode.ACTIVE)
-                        .install();
+                ServiceBuilder<?> builder = target.addService(IOServices.WORKER.append("default"));
+                Consumer<XnioWorker> workerConsumer = builder.provides(IOServices.WORKER.append("default"));
+                builder.setInstance(new WorkerService(workerConsumer, null, Xnio.getInstance().createWorkerBuilder().setWorkerIoThreads(2)));
```

Comment:
Instead of null can this pass in a simple Supplier of some simple ExecutorService?

That's out of scope for this really but it will help clean up test noise.